### DISCUSSION TITLE
[Debian/Ubuntu etc.] Is there still a problem with too-old packages?

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1131] [Team Posts]
Is there still a problem with old packages? I'm running a different distro base and can't test.
[https://launchpad.net/ubuntu/+source/muse](url)
Looks like there's 3.0.2
Is all well in that world?

**Comment by spamatica** [comment]
Den lör 7 dec. 2019 kl 21:10 skrev Tim <notifications@github.com>:

> Is there still a problem with old packages? I'm running a different distro
> base and can't test.
> https://launchpad.net/ubuntu/+source/muse <http://url>
> Looks like there's 3.0.2
> Is all well in that world?
>

Yay, you are right. Ubuntu seems to have 3.02 from the 19.04 release and
Debian has it in all relevant releases, stable, unstable and testing.
Slightly worse for Ubuntu since 18.04 is the long term support release and
it doesn't have it yet. (Verified by installing as i have 18.04 on this
machine)
